services/forms/repo_form.go

```diff
@@ -599,6 +599,32 @@ func (f *MergePullRequestForm) Validate(req *http.Request, errs binding.Errors)
 	return middleware.Validate(errs, ctx.Data, f, ctx.Locale)
 }
 
+// SetDefaults for options who n
+func (f *MergePullRequestForm) SetDefaults(pr *models.PullRequest) {
+	if len(f.Do) == 0 {
```

Comment:
```suggestion
	if f.Do == "" {
```

it's fine to use modern syntax (and for `len(f.MergeTitleField)` lines below)

Reply:
And the comment ` // SetDefaults for options who n ` seems broken